pkg/jsonmessage/jsonmessage.go

```diff
@@ -99,6 +99,7 @@ type JSONMessage struct {
 	ID              string        `json:"id,omitempty"`
 	From            string        `json:"from,omitempty"`
 	Time            int64         `json:"time,omitempty"`
+	TimeNano        int64         `json:"timeNano,omitempty"`
```

Comment:
but the existent field is not set anymore, is it? I don't know how I feel about a field that's never set.